### PR TITLE
Update fasta w.r.t. submitted versions

### PR DESCRIPTION
**Pull request by bradcray**
Belatedly, this adds the fasta#2 version that we submitted to the CLBG in December to the studies/shootout/submitted/ directory.  It also removes fastaredux from that directory and from the release, since the CLBG isn't tracking fastaredux any longer.  In support of this, it:

* updates the submitted fasta.graph file
* removes fastaredux from the release README and Makefile
* updates the shootout/submitted/README.md to include both fastas and both pidigits and no fastaredux

### CODE REVIEW

**Comment by bradcray** [comment]
@ronawho, would you mind reviewing?

File: test/studies/shootout/submitted/fasta2.execopts [hunk 1]
```diff
@@ -0,0 +1 @@
+--blockSize=4
```
Comment (ronawho) [inline test/studies/shootout/submitted/fasta2.execopts]: Do we need/want this for the submitted version?

Comment (bradcray) [inline test/studies/shootout/submitted/fasta2.execopts]: Without it, the correctness test is serialized which sometimes makes correctness errors in the parallel strategy not get caught until performance testing (if they're caught there).  So to the extent that the two versions might diverge from one another over time, I think it makes sense to correctness test it in this way, yes.  Let me know if you disagree.

Comment (ronawho) [inline test/studies/shootout/submitted/fasta2.execopts]: ok, no objections

**Pull request merged**